challenges/objects-arrays.js

```diff
@@ -6,6 +6,29 @@
   object name, diet, weight, length, period
 */
 
```

Comment:
Another way to write this could be
class Dino {
  constructor(name, diet, weight, length, period) {
    this.name = name;
    this.diet = diet;
    this.weight = weight;
    this.length = length;
    this.period = period;
  }
}


let rex = new Dino('Tyranosaurus', 'Carnivorous', etc)